webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue

```diff
@@ -182,6 +183,12 @@ export default {
         return null;
       },
     },
+    filterStyle: {
+      type: Boolean,
+      default: function () {
+        return false;
+      },
```

Comment:
When it's boolean, you can avoid declaring the default value as a function
```suggestion
      default: false,
```

Reply:
done